boards/arc/em_starterkit/doc/board.rst

```diff
@@ -151,6 +167,25 @@ setup the board correctly.
 * If you want to know more about how to use this board, you can take a look
   at the `ARC EM Starter Kit User Guide`_.
 
+Set up Zephyr Software
+======================
+
+Since there are different firmware versions of EM Starter Kit, you need to
+choose the proper firmware version supported in Zephyr.
+If you select one core configuration in EM Starter Kit, you need to change
+the default EM Starter Kit board default configuration file
+:file:`/boards/arc/em_starterkit/em_starterkit_defconfig`.
+
+For example, if you choose EM Starter Kit 2.2 EM11D, you need to change the
```

Comment:
since these line numbers may change, I'd recommend just saying, 

     For example, if you choose EM Starter Kit 2.2 EM11D, you need to change the
     ``em_starterkit_defconfig`` file to say: